tests/console/yast2_kdump.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Test scenario which configures Kdump with a YaST module
+# and checks configuration without rebooting the system.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration;
+use kdump_utils;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    select_console('root-console');
+    my @conf_files = @{get_test_suite_data()->{kdump}};
+
+    # install kdump by adding additional modules
+    add_suseconnect_product('sle-module-desktop-applications');
+    add_suseconnect_product('sle-module-development-tools');
```

Comment:
I was under the impression that you don't need these too products. The server is in textmode and the test doesn't attempt to read coredump. What am I missing?

Reply:
It is needed, without them, we cannot zypper the package.